atomic_reactor/tasks/common.py

```diff
@@ -122,7 +132,32 @@ def run(self, *args, **kwargs):
             else:
                 signal.signal(signal.SIGTERM, self.throw_task_canceled_exception)
 
-            result = self.execute(*args, **kwargs)
+            reactor_config_path = self._params.config_file or REACTOR_CONFIG_FULL_PATH
+            conf = Configuration(config_path=reactor_config_path)
+
+            userdata = self._params.user_params.get('userdata', None)
```

Comment:
instead of userdata, create some new field, 'opentelementry' or something which will contain both traceparentid & otel url

Reply:
created `opentelemetry_info`